composer/loggers/file_logger.py

```diff
@@ -61,9 +63,11 @@ class FileLogger(LoggerDestination):
 
 
     Args:
-        filename (str): File to log to.
-            Filepaths should be specified relative to the
-            :mod:`~.composer.utils.run_directory`.
+        filename (str): Filepath to log to, relative to the :mod:`~.composer.utils.run_directory`.
+        capture_stdout (bool, optional): If ``True`` (the default), writes to ``stdout`` will be included in
+            ``filename``.
+        capture_stderr (bool, optional): If ``True`` (the default), writes to ``stderr`` will be included in
+            ``filename``.
```

Comment:
Definitely emphasize that the default for these args is `True`, not `False` as is often assumed for optional boolean parameters.

Reply:
Added the standard (default: ``True``) to the end.